src/components/track-changes/TrackChangesPanel.tsx

```diff
@@ -46,6 +46,16 @@ export const TrackChangesPanel: React.FC = () => {
 
   const { changeSet } = trackState || {}
 
+  const changes = useMemo(
+    () =>
+      changeSet?.groupChanges.filter(
+        (c) =>
+          c.dataTracked.status === CHANGE_STATUS.pending &&
+          c.dataTracked.operation !== CHANGE_OPERATION.reference
```

Comment:
I would say that these details should be covered inside of ChangeSet and hidden by the abstraction that I suggested earlier. We may have a lot of different types of changes and this UI component should reason about how which one are displayable and which are groups and which are not. 
I would suggest that when we do "changeSet?.pending" it will already filter away all the changes that shouldn't be use here or in any other UI component.

Reply:
updated